dawn/renderer/components/peripherals/Motor.js

```diff
@@ -1,37 +1,36 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import _ from 'lodash';
-import { ProgressBar } from 'react-bootstrap';
 import numeral from 'numeral';
 
+import { ProgressBar } from 'react-bootstrap';
+
 /**
- * A component representing a motor.
+ *  Motor Component
  */
 const Motor = props => (
```

Comment:
Instead of `props`, you can probably just spread out the attributes: `{ id, param, device_name }`.